### PR TITLE
Updating Comments in EIP 1271

### PR DESCRIPTION
**Pull request by Muhammad-Altabba**
Updating the comments according to the early change of the parameter `_data` to be `_hash`.

### CODE REVIEW

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 1271 requires approval from one of (@izqui, @phabc, @abandeali1, @catageek, @shrugs, @frangio)

File: EIPS/eip-1271.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ contract ERC1271 {
 
   /**
    * @dev Should return whether the signature provided is valid for the provided data
-   * @param _data Arbitrary length data signed on the behalf of address(this)
-   * @param _signature Signature byte array associated with _data
+   * @param _hash hash of the data that is signed on the behalf of address(this)
+   * @param _signature Signature byte array of the data
```
Comment (MicahZoltu) [inline EIPS/eip-1271.md:41]: Changing from "associated with the _data" to "of the data" seems like a normative change as those two statements have different meanings.  Is it established elsewhere in this EIP that the signature is specifically of the data itself and not just associated with it?

Comment (Muhammad-Altabba) [reply EIPS/eip-1271.md]: You are correct. I changed it back to be "associated with" but with the _hash.

**Comment by MicahZoltu** [comment]
You need an approval from one of the authors.  Editors (like me) are just here to make sure things follow the rules, are structurally sound, and ideally are well defined.

**Comment by Muhammad-Altabba** [comment]
Hello @izqui, @PhABC, @abandeali1, @catageek, @shrugs, @frangio
Could you please review this MR?
Thanks,

**Review by shrugs — COMMENTED** [review]
lgtm

**Pull request closed without merging**